mediathread/settings_shared.py

```diff
@@ -318,6 +318,8 @@ def default_url_processor(url, label=None, request=None):
     'embed_tool_id': 'mediathread',
 }
 
+SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTOCOL', 'https')
```

Comment:
Okay, I've verified via the Django Debug Toolbar/Headers panel on staging that the header being sent through via nginx is: X-Forwarded-Protocol.

From what I can gather, [both X-Forwarded-Protocol and X-Forwarded-Proto](http://stackoverflow.com/questions/16042647/whats-the-de-facto-standard-for-a-reverse-proxy-to-tell-the-backend-ssl-is-used) are in use. And, there's an [RFC to standardize headers](https://tools.ietf.org/html/rfc7239).

So, for the moment, I think it's safe & fine to stick with X-Forwarded-Protocol.